schemas/activity/cranialWindowPreparation.schema.tpl.json

```diff
@@ -14,25 +12,25 @@
       ]
     },
     "dimension": {
-      "_instruction": "Add the dimension of the cranial window.",
+      "_instruction": "Enter the dimension of the cranial window by defining its mathematical shape.",
       "_embeddedCategories": [
         "mathematicalShapes"
       ]
     },
     "input": {
-      "_instruction": "Add the state of the subject before the cranial window was created.",
+      "_instruction": "Add the state of the subject which received the cranial window during this activity.",
```

Comment:
"Add the state of the subject which received the cranial window **before** this activity."

Reply:
I would agree with that suggestion